src/Kdyby/Translation/PrefixedTranslator.php

```diff
@@ -49,6 +50,14 @@ public function __construct($prefix, ITranslator $translator)
 
 	public function translate($message, $count = NULL, $parameters = array(), $domain = NULL, $locale = NULL)
 	{
+		if (Utils\Strings::startsWith($message, '../')) {
```

Comment:
mabye i write it wrong. it is not relative but absolute. The point is to skip the prefix when you use prefixed translator